src/schemathesis/models.py

```diff
@@ -39,6 +40,8 @@ class Case:
     body: Optional[Body] = attr.ib(default=None)  # pragma: no mutate
     form_data: Optional[FormData] = attr.ib(default=None)  # pragma: no mutate
 
+    feedback: "Feedback" = attr.ib(default=None)
```

Comment:
It shouldn't be visible in the repr, and it would be nice to have it as a private attribute